src/main.cpp

```diff
@@ -101,6 +101,38 @@ static void printInfo()
     std::cout << vc4c::to_string<std::string>(infoString, "; ") << std::endl;
 }
 
+/*
+ * parse options with parameter like xxx=n
+ * if invalid parameter are passed,
+ */
+Optional<int> parseIntOption(std::string name, std::string input)
+{
+    std::stringstream ss(input);
+    std::string buffer;
+    std::getline(ss, buffer, '=');
+    if(buffer == name)
+    {
+        std::getline(ss, buffer, '=');
+        if(name == buffer)
```

Comment:
Why do you re-test for the name?
I think it would be more obvious what is tested here if you reset buffer to an empty string and here test for the buffer still being empty.

Reply:
If `ss` is `--fcombine-load-threshold` (just this, not including '='), `stringstream` reach the end and second `std::getline` do nothing. So `name == buffer` is needed here.
I add comments about this.